mp4parse/src/lib.rs

```diff
@@ -23,6 +24,10 @@ mod tests;
 // Arbitrary buffer size limit used for raw read_bufs on a box.
 const BUF_SIZE_LIMIT: u64 = 1024 * 1024;
 
+// Tags for elementary stream description
+const ESDESCR_TAG: u8           = 0x03;
+const DECODER_CONFIG_TAG: u8    = 0x04;
```

Comment:
These are only used inside `read_esds`, so move them into the scope of the function.